src/LibraryViewExtensionMSWebBrowser/LibraryViewController.cs

```diff
@@ -563,6 +569,16 @@ private void InitializeResourceProviders(DynamoModel model, LibraryViewCustomiza
             layoutProvider = new LayoutSpecProvider(customization, iconProvider, "Dynamo.LibraryViewExtensionMSWebBrowser.web.library.layoutSpecs.json");
         }
 
+        /// <summary>
+        /// This method will execute the action of moving the Guide to the next Step (it is triggered when a specific html div that contains the package is clicked).
+        /// </summary>
+        internal void MoveToNextStep()
+        {
+            dynamoViewModel = (dynamoWindow.DataContext as DynamoViewModel);
+            Guide executingGuide = dynamoViewModel.MainGuideManager.CurrentGuide;
+            executingGuide.NextStep(executingGuide.CurrentStep.Sequence);
+        }
```

Comment:
I would not expect a function like this in `LibraryViewController` since LibraryViewController should not know anything about Dynamo guide.. Shall we change this to event based?

Reply:
@QilongTang There was an update for the NextStep and PrevStep events so after this change we will be allowed to call the NextStep event without passing arguments and can be called from the LibraryViewController.cs (now we will using events instead).
See the commit: https://github.com/DynamoDS/Dynamo/pull/12312/commits/0397fe711f7a87f84e45c77e4fc9ddd615ff2c2b